app/routes/docs.tsx

```diff
@@ -24,29 +24,6 @@ export default function () {
               <NavLink key="." to="." end>
                 About GCN
               </NavLink>,
-              <>
```

Comment:
What is your rationale for the new order for the table of contents? I notice it's in alphabetical order, but I think that a _logical_ order would be preferable.

Reply:
OK. I don't think that the alphabetical order will be a durable choice, but it's fine for now.